package-lock.json

```diff
@@ -2973,19 +2973,19 @@
       }
     },
     "node_modules/@mui/material": {
-      "version": "5.4.4",
-      "resolved": "https://registry.npmjs.org/@mui/material/-/material-5.4.4.tgz",
-      "integrity": "sha512-VDJC7GzO1HTFqfMe2zwvaW/sRhABBJXFkKEv5gO3uXx7x9fdwJHQr4udU7NWZCUdOcx9Y0h3BsAILLefYq+WPw==",
+      "version": "5.5.0",
```

Comment:
@philipjackson WTF? This version literally doesn't exist on npm registry!

Reply:
Can you change the package.json to just be the first rule (i.e. delete the `||` and onwards? @SilviaZeta you will need to do the same in CURATE is seems.

Reply:
(My WTF is at npm not at you lol)

Reply:
Now there's a `csstype` error